x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/tour_config.ts

```diff
@@ -118,13 +116,25 @@ const alertsCasesConfig: StepConfig[] = [
         defaultMessage: `In addition to the alert, you can add any relevant information you need to the case.`,
       }
     ),
-    anchor: `[data-test-subj="create-case-flyout"]`,
+    anchor: `[data-test-subj="create-case-flyout"] [data-test-subj="caseTitle"] label`,
     anchorPosition: 'leftUp',
+    offset: 20,
     dataTestSubj: getTourAnchor(5, SecurityStepId.alertsCases),
-    hideNextButton: true,
   },
 ];
 
+export const sampleCase = {
+  title: i18n.translate('xpack.securitySolution.guided_onboarding.tour.createCase.title', {
+    defaultMessage: `Malicious signal detected`,
+  }),
+  description: i18n.translate(
+    'xpack.securitySolution.guided_onboarding.tour.createCase.description',
+    {
+      defaultMessage: `This is a very unfortunate case of a malicious signal. I've seen it do horrible things. My oh my what a terror, run for your lives!`,
```

Comment:
```suggestion
      defaultMessage: `This is where you'd document a malicious signal. You can include whatever information is relevant to the case and would be helpful for anyone else that needs to read up on it.`,
```

Reply:
I was close

Reply:
@kellyemurphy maybe we can also show off that it supports markdown?

Reply:
Yes, we can add a line "Markdown formatting is supported."